### ISSUE TITLE
Feature Request: Failing SMART reports include WCAG / EPUB Conformance Specification Version

### ISSUE THREAD

**Issue by clapierre** [daisy/ace-smart#7]
Currently when you Pass a SMART report it tells you this information: 
Conformance:	Pass: EPUB Accessibility 1.1 - WCAG 2.1 Level AA

However when you fail you just see

Conformance:	Failed

So one has no idea which WCAG level this report was evaluated against.
Would be great to include this information in all reports regardless if it passes or fails.
Thanks!

**Comment by mattgarrish** [comment]
Thanks @clapierre!

How about separating the fields? Maybe like this:

Evaluated against:&#160;&#160;&#160;&#160;&#160;&#160;&#160;EPUB Accessibility 1.X ...
Conformance:&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;&#160;Pass/Fail/Incomplete

(Edit: changed "evaluated to" to "evaluated against")

**Comment by mattgarrish** [comment]
I've got this working now on the dev server if you want to give it a test run @clapierre.

**Comment by clapierre** [comment]
Works great Matt! Feel free to push this to production as you see fit.  BTW I will have a new GCA SMART JSON file to upload on Mar 1.  
Thanks

**Comment by mattgarrish** [comment]
Thanks Charles. It's live now. Just send me the other file whenever it's ready.
